security/handler/security-handler-saml/src/main/java/org/codice/ddf/security/handler/saml/SAMLAssertionHandler.java

```diff
@@ -53,12 +57,21 @@
      */
     public static final String AUTH_TYPE = "SAML";
 
+    public static final String SAML_HEADER_NAME = "Authorization";
+
     private static final Logger LOGGER = LoggerFactory.getLogger(SAMLAssertionHandler.class);
 
     public SAMLAssertionHandler() {
         LOGGER.debug("Creating SAML Assertion handler.");
     }
 
+    public static String decodeSaml(String encodedToken) throws IOException {
```

Comment:
@pklinef I added these methods in in before we decided to change RestSecurity to send headers. (RestSecurity previously only had 1 encodeMethod that took a token, not a string) When we decided to change RestSecurity, I added in the new encode and decode methods and forgot to update this class to use RestSecurity methods instead. I'll have it fixed in my next update that will be up momentarily.